Packs/Malwarebytes/Integrations/Malwarebytes/Malwarebytes_test.py

```diff
@@ -1159,23 +1159,23 @@ def test_list_endpoint_info_hostname(requests_mock, mocker):
         'Malwarebytes.Endpoint(val.Hostname == obj.Hostname)': {
             'Hostname': MOCK_ASSETS_RESPONSE['host_name'],
             'IPAddress': MOCK_ASSETS_RESPONSE['nics'],
-            'Domain': MOCK_ASSETS_RESPONSE['domain_name'],
+            'Domain': MOCK_ASSETS_RESPONSE.get('domain_name'),
             'MACAddress': MOCK_ASSETS_RESPONSE['nics'][0]['mac_address'],
             'OS': MOCK_ASSETS_RESPONSE['os_info']['os_platform'],
             'OSVersion': MOCK_ASSETS_RESPONSE['os_info']['os_version'],
-            'Model': MOCK_ASSETS_RESPONSE['computer_info']['model'],
-            'Memory': MOCK_ASSETS_RESPONSE['memory'],
+            'Model': str(MOCK_ASSETS_RESPONSE.get('computer_info', {}).get('model')),
+            'Memory': str(MOCK_ASSETS_RESPONSE.get('memory')),
             'Assets': MOCK_ASSETS_RESPONSE
         },
         'Endpoint(val.Hostname == obj.Hostname)': {
             'Hostname': MOCK_ASSETS_RESPONSE['host_name'],
             'IPAddress': MOCK_ASSETS_RESPONSE['nics'][0]['ips'][0],
-            'Domain': MOCK_ASSETS_RESPONSE['domain_name'],
+            'Domain': MOCK_ASSETS_RESPONSE.get('domain_name'),
             'MACAddress': MOCK_ASSETS_RESPONSE['nics'][0]['mac_address'],
             'OS': MOCK_ASSETS_RESPONSE['os_info']['os_platform'],
             'OSVersion': MOCK_ASSETS_RESPONSE['os_info']['os_version'],
-            'Model': MOCK_ASSETS_RESPONSE['computer_info']['model'],
-            'Memory': MOCK_ASSETS_RESPONSE['memory']
+            'Model': str(MOCK_ASSETS_RESPONSE.get('computer_info', {}).get('model')),
+            'Memory': str(MOCK_ASSETS_RESPONSE.get('memory')),
```

Comment:
Why did you wrap the returned result with a string?
If as stated in the release notes the problem was MOCK_ASSETS_RESPONSE['memory']. the fix you made with MOCK_ASSETS_RESPONSE.get('memory') should do the trick since the code won't fail on KeyError if the memory field returns empty. (The get function would not fail and return a None value).

Reply:
Oh I didn't know the string conversion can prevent users to get the dict. Then i can remove it. Thanks